.circleci/main.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - run:
           no_output_timeout: 20m
           command: |-
-            GOMAXPROCS=2 KUBECONFIG=/dev/null PACH_CONFIG=/dev/null \
+            GOMAXPROCS=8 KUBECONFIG=/dev/null PACH_CONFIG=/dev/null \
```

Comment:
did you try any larger values for GOMAXPROCS? we might be able to bump this up since we are using an xlarge machine

Reply:
Yup, I tried 16 but I didn't see any improvements in time. I think we're bound by our CI node here